experimental/torch_xla2/test/test_ops.py

```diff
@@ -224,6 +224,7 @@ def run_export_and_compare(testcase,
               rtol=rtol,
               equal_nan=equal_nan, check_output=check_output)
         else:
+          print(func.name, "res: ", res, "res2: ", res2)
```

Comment:
thanks, do we want to keep these `print` code here?

Reply:
No. Still WIP.

Reply:
removed.